domains/icvn.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "A minecraft server",
+  "repo": "https://github.com/HaiYTB/register",
+  "owner": {
+    "username": "HaiYTB",
+    "email": "haiwt2xbox@gmail.com"
+  },
+  "record": {
+    "A": ["103.229.52.14:25908"]
```

Comment:
Invalid. Ports aren't allowed in A records.

Reply:
What should I do?